services/src/main/java/org/exoplatform/webconferencing/notification/utils/NotificationConstants.java

```diff
@@ -0,0 +1,22 @@
+package org.exoplatform.webconferencing.notification.utils;
```

Comment:
header is mssing

Reply:
It is not the good one. LGPL is for meeds-io organization. For exoplatform org it is AGPL
Please update